### PR TITLE
Fix dockerfile

### PR DESCRIPTION
**Pull request by bartv**
# Description

There was something wrong in my previous fix. I tested it locally but probably made mistake. This one succeeded on jenkins.

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [ ] Attached issue to pull request
- [ ] Changelog entry
- [ ] Type annotations are present
- [ ] Code is clear and sufficiently documented
- [ ] No (preventable) type errors (check using make mypy or make mypy-diff)
- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Correct, in line with design
- [ ] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )

# Reviewer Checklist:

- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Code is clear and sufficiently documented
- [ ] Correct, in line with design

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -17,7 +17,10 @@ RUN yum install -y \
 		make \
 		yarn \
 		postgresql \
-		libpq-devel
+		postgresql-devel
+
+ENV PATH "${PATH}:/usr/pgsql-10/bin"                                                                 
+ENV PG_CONFIG_DIR /usr/pgsql-10/bin
```
Comment (arnaudsjs) [inline Dockerfile:23]: Why is this required? That directory doesn't exist in the container as far as I can see. Can you add a comment?

**Review by arnaudsjs — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (bartv) [reply Dockerfile:23]: It exists once you install postgresql-devel. This is required for pscopg2 install to succeed

Comment (arnaudsjs) [reply Dockerfile:23]: Strange. In my container it isnt'.

```
[root@71ec33266e71 /]# yum list installed|grep -i postgres
postgresql.x86_64                       9.2.24-4.el7_8               @base      
postgresql-devel.x86_64                 9.2.24-4.el7_8               @base      
postgresql-libs.x86_64                  9.2.24-4.el7_8               @base      
[root@71ec33266e71 /]# rpm -ql postgresql|grep -i pgsql-10
[root@71ec33266e71 /]# rpm -ql postgresql-devel|grep -i pgsql-10
[root@71ec33266e71 /]# rpm -ql postgresql-libs|grep -i pgsql-10
[root@71ec33266e71 /]#
```

**Review by arnaudsjs — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (bartv) [reply Dockerfile:23]: hmmm, but this made the build work. Maybe I should try without the env then.

https://jenkins.inmanta.com/job/integration-tests/job/inmanta-core-docker/default/

**Comment by bartv** [comment]
Works without  the env variables, so I removed them. Thanks for catching this!

**Pull request merged**